docs/meshstack.openshift.md

```diff
@@ -0,0 +1,6 @@
+---
+id: meshstack.openshift
+title: OpenShift
+---
+
+Meshstack supports management of RedHat OpenShift platforms. OpenShift has a [Kubernetes](meshstack.k8s.md) core and provides additional services. It is available in both Open Source flavors as well as enterprise offerings by RedHat.
```

Comment:
d: Should we also add the short remark, as for AWS, that we support SSO, project replication and user management?

Reply:
yes, good point